lightningd/plugin_control.c

```diff
@@ -27,32 +27,13 @@ static struct command_result *plugin_dynamic_list_plugins(struct command *cmd,
 		json_add_string(response, "name", p->cmd);
 		json_add_bool(response, "active",
 		              p->plugin_state == INIT_COMPLETE);
+		json_add_u32(response, "state", p->plugin_state);
```

Comment:
This is the numeric state which won't be very helpful for users looking at it (especially should we ever add intermediate states), can we have a mapping from enum to a human readable string here?

Reply:
Oops, this was a debugging thing I forgot to remove!!